packages/ui-renderer/src/components/textarea.tsx

```diff
@@ -29,11 +29,15 @@ const StyledIgnoreKeys = styled(IgnoreKeys)({
 })
 
 export function EditorTextarea(
-  props: Omit<TextareaAutosizeProps, 'as' | 'ref'>
+  props: Omit<TextareaAutosizeProps, 'as' | 'ref'> & {
+    inputRef?:
+      | React.RefObject<HTMLTextAreaElement>
+      | ((node: HTMLTextAreaElement) => void)
+  }
```

Comment:
from the usage below I'd guess, that inputRef is already specified correctly in `TextareaAutosizeProps`

Reply:
👍